exercises/nth-prime/example.tt

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env ruby
+gem 'minitest', '>= 5.0.0'
+require 'minitest/autorun'
+require_relative 'nth_prime'
+
+# Test data version:
+# <%= sha1 %>
+# Rubocop directives
+# rubocop:disable Style/NumericLiterals
+#
+class NthPrimeTest < Minitest::Test<% test_cases.each do |test_case| %>
+  def <%= test_case.name %><% if test_case.skipped? %>
+    skip<% end %>
+    assert_equal <%= test_case.expected %>, <%= test_case.actual %>
+  end
```

Comment:
It would be good to remove the conditional from the template.
See how https://github.com/exercism/xruby/blob/master/lib/rna_transcription_cases.rb handles this.